src/pages/Feed.js

```diff
@@ -1,40 +1,43 @@
-// import AddPostForm from "../components/posts/AddPostForm.js";
 import PostList from "../components/posts/PostList.js";
 import { useState, useEffect } from "react";
 
-function FeedPage() {
+function FeedPage(props) {
+  const refreshPost = props.refreshPost;
   const [isLoading, setIsLoading] = useState(true);
   const [loadedPosts, setLoadedPosts] = useState([]);
 
-  useEffect(()=>{
+  useEffect(() => {
+    props.setRefreshPost(false);
     setIsLoading(true);
-    fetch("https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts.json")
+    fetch(
+      "https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts.json"
+      )
       .then((response) => {
         return response.json();
       })
       .then((data) => {
-        const posts = []
-
-        for(const key in data){
+        const posts = [];
+        
+        for (const key in data) {
           const post = {
             id: key,
-            ...data[key]
+            ...data[key],
           };
-
-          posts.push(post)
+          
+          posts.push(post);
```

Comment:
There might be a way to use `map` here to make this code look cleaner though that is likely personal taste so up to you

Reply:
true